research-software-directory/RSD-as-a-service
# fix: pages have default ordering, remove ordering from navigation links

## Default ordering for software overview pages, hide request counts for communities

Changes proposed in this pull request:

* Remove the verious ordering query parameters from various links
* When no, or an invalid, ordering is in the URL, a default ordering is used
* Hide the request counts from the community overview page when not an admin
* Users can only see approved software on a community page (and in the database)

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Enable the community module
* Using the navigation bar, visit software and projects, notice there is no order query parameter, but the filter modal on the left does show an ordering
* Same when visiting a community
* Same when visiting an organisation
* On the community overview pages, the request counts should only be shown for RSD admins

Closes #1256

PR Checklist:
